var/spack/repos/builtin/packages/metabat/package.py

```diff
@@ -13,6 +13,8 @@ class Metabat(SConsPackage):
     homepage = "https://bitbucket.org/berkeleylab/metabat"
     url      = "https://bitbucket.org/berkeleylab/metabat/get/v2.12.1.tar.gz"
 
+    version('2.14',   sha256='d43d5e91afa8f2d211a913739127884669516bfbed870760597fcee2b513abe2')
+    version('2.13',   sha256='aa75a2b62ec9588add4c288993821bab5312a83b1259ff0d508c215133492d74')
     version('2.12.1', sha256='e3aca0656f56f815135521360dc56667ec26af25143c3a31d645fef1a96abbc2')
     version('2.11.2', sha256='9baf81b385e503e71792706237c308a21ff9177a3211c79057dcecf8434e9a67')
 
```

Comment:
Hi @erimar it looks like this triggered a secondary check: with https://github.com/spack/spack/pull/11115 packages are required to implement:

```
setup_build_environment(self, env)
```

vs.

```
setup_environment(spack_env, run_env)
```

this package should have been addressed as part of that PR but slipped through apparently. If you have the chance could you update the method?

Reply:
@erimar77 - I accidentally pinged `erimar` (sorry `erimar` if this summons you here without context).

Reply:
@scheibelp #11115 didn't update any packages. #13411 will update ~40 of them though, so that's a step in the right direction.

Reply:
@scheibelp Just for context - the changes in #11115 were designed to have a backward compatible fallback and to not touch most of the packages. The idea was to allow for a soft transition to the new API using:
- A test that forces the removal of every deprecated API call in a PR (otherwise it fails)
- A test (currently marked `xfail`) that ensures no package has any deprecated method

Part of the rationale is that we'll spread news on the new API via reviews + old APIs are still working. Once we are ready to convert every package we can remove the first test and unmark the second from being `xfail`.